en/core-libraries/text.rst

```diff
@@ -38,6 +38,52 @@ of a ``View``, use the ``Text`` class::
         }
     }
 
+
+Transliterate Strings
```

Comment:
Might want to use 'Convert Strings into ASCII'. That will be a more search/skim read friendly term.